### PR TITLE
be explicit about permissions, avoid images with too restrictive permissions because of umask

### PR DESCRIPTION
**Pull request by mattf**

### CODE REVIEW

**Comment by mattf** [comment]
@elmiko ptal

File: Dockerfile [hunk 1]
```diff
@@ -7,4 +7,6 @@ ADD . /go/src/github.com/redhatanalytics/oshinko-rest
 WORKDIR /go/src/github.com/redhatanalytics/oshinko-rest
 RUN make install
 
+RUN chmod a+rwX -R .
```
Comment (elmiko) [inline Dockerfile]: this totally works, do you think we should come back at some point and restrict the execute perms to only hit the files we care about?

Comment (elmiko) [inline Dockerfile]: looks like only these 2 need execute:
* `/go/bin/oshinko-rest-server`
* `/go/src/github.com/redhatanalytics/oshinko-rest/tools/start_server.sh`

Comment (mattf) [inline Dockerfile]: 'X' only sets execute on files that already have execute, which differs from 'x' that always sets execute

Comment (elmiko) [inline Dockerfile]: ah, gotcha. i thought it set on all non-directory files. cool, lgtm

Comment (elmiko) [inline Dockerfile]: do we need to set it explicitly on the start script?

Comment (mattf) [inline Dockerfile]: afaict the mode is recorded in git as executable

Comment (elmiko) [inline Dockerfile]: cool, sounds good

**Pull request merged**